src/Adapter/Configuration.php

```diff
@@ -124,21 +141,25 @@ public function get($key, $default = null)
      *
      * @param string $key
      * @param mixed $value
+     * @param ShopConstraint|null $shopConstraint
      * @param array $options Options
```

Comment:
Please deprecate this parameter

Reply:
Which one? `$options` ?

Reply:
It is still used for now.. Does it mean we keep using it for now but it will disappear (along with its feature) in the next version?